docs/content/guides/developer/nft/nft-soulbound.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Soulbound NFT Example
+description: An example using Sui Move struct abilities and the Sui Framework's `transfer` module to make a NFT soulbound (non-transferable).
+keywords: [ERC-721, NFT, Soulbound]
+---
+
+A soulbound NFT is an NFT that is non-transferable. Once a NFT is minted to a Sui account, the NFT is "bounded" to that account and cannot be transferred. This implementation leverages the custom logic of the Sui framework's transfer functions. The [`sui::transfer` module](/references/framework/sui-framework/transfer.md) contains two functions that transfers objects: [`transfer::transfer`](/references/framework/sui-framework/transfer.md#function-transfer) and [`transfer::public_transfer`](/references/framework/sui-framework/transfer.md#function-public_transfer). 
+
+Typically, when defining new NFTs or object types on Sui, you don't need to create a transfer function because the Sui Framework offers `transfer::public_transfer` which anyone can use to transfer objects. However, `transfer::public_transfer` requires transferred objects have the `key` and `store` ability. Therefore, if you define a new NFT type that has the `key` ability, meaning it is a Sui object, but not the `store` ability, the holders won't be able to use `transfer::public_transfer`. This results in a soulbound NFT.
+
+It is also possible to create custom transfer logic for NFTs on Sui. The `transfer::transfer` function has custom rules performed by the Sui Move bytecode verifier that ensures that the transferred objects is an object defined in the module where transfer is invoked. While removing the `store` ability from a struct definition makes `transfer::public_transfer` unusable, `transfer::transfer` can still be used as long as it's used in the module that defined that object's type. This allows for the module owner to provide custom transfer logic for their soulbound NFTs. 
```

Comment:
```suggestion
It is also possible to create custom transfer logic for NFTs on Sui. The `transfer::transfer` function has custom rules performed by the Sui Move bytecode verifier that ensures that the transferred objects are defined in the module where transfer is invoked. While removing the `store` ability from a struct definition makes `transfer::public_transfer` unusable, `transfer::transfer` can still be used as long as it's used in the module that defined that object's type. This allows for the module owner to provide custom transfer logic for their soulbound NFTs. 
```
Point of this suggestion is to make the plural matchy matchy. You could also choose to make "the transferred objects" singular instead.